examples/xlio_ultra_api_ping_pong.c

```diff
@@ -37,20 +53,38 @@
 /* Application configuration */
 struct app_config {
     bool is_server;
-    const char *ip;
-    const char *bind_ip; /* Local IP to bind client socket to */
+    const char *ip; /* Server IP to bind to or to connect to for server and client respectively */
+    const char *client_ip; /* Local IP to bind client socket to */
     unsigned short port;
     int ping_count;
     int timeout_seconds; /* Application timeout in seconds (0 = no timeout) */
 };
 
+/* RX data accumulator for message assembly */
+struct rx_accumulator {
```

Comment:
we need to be here more descriptive for example code.
A.I. suggestion:
`/* RX data accumulator for message assembly from TCP stream.
 * TCP doesn't preserve message boundaries, so we accumulate data until
 * we find the protocol delimiter (newline character).
 */
struct rx_accumulator {
    char *data;     /* Buffer holding accumulated bytes from RX callbacks */
    size_t len;     /* Number of valid bytes currently in 'data' */
    size_t capacity; /* Total allocated size of 'data' buffer */
};`

you don't have to follow the exact suggestion - but the comments should be "dummy-proof" :)